test_scripts/Polices/Related_HMI_API/ATF_GetUserFriendlyMessage_with_Language.lua

```diff
@@ -0,0 +1,67 @@
+
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies]: SDL.GetUserFriendlyMessage ("language")
+-- [HMI API] SDL.GetUserFriendlyMessage request/response
+--
+-- Description:
+-- 1. Precondition: SDL is started, HMI is started, App is registered
+-- 2. Steps: Activate App, in SDL.GetUserFriendlyMessage parameter "language" should be present
+--
+-- Expected result:
+-- PoliciesManager must respond with the information from the sub-section with the name correspondent to "language" value
+---------------------------------------------------------------------------------------------
+ 
+--[[ General configuration parameters ]]
+Test = require('connecttest')
+require('cardinalities')
+require('mobile_session')
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+ 
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local testCasesForBuildingSDLPolicyFlag = require('user_modules/shared_testcases/testCasesForBuildingSDLPolicyFlag')
+require('user_modules/AppTypes')
+ 
+--[[ General Precondition before ATF start ]]
+testCasesForBuildingSDLPolicyFlag:Update_PolicyFlag("EXTENDED_POLICY", "EXTERNAL_PROPRIETARY")
+testCasesForBuildingSDLPolicyFlag:CheckPolicyFlagAfterBuild("EXTENDED_POLICY","EXTERNAL_PROPRIETARY")
+commonSteps:DeleteLogsFileAndPolicyTable()
+ 
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+commonSteps:DeleteLogsFileAndPolicyTable()
+ 
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:ActivateApp_with_language()
```

Comment:
Please add to name of functions in this section TestStep_ e.g.
function Test:TestStep_ActivateApp_with_Language_Message